apps/web/src/components/CareLocationSelector.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Dropdown } from './Dropdown';
+import { getCareLocations } from '../services/carelocations';
+
+interface UnitProps {
+  id: string;
+  unitName: string;
+}
+
+export const CareLocationSelector = () => {
+  const [careLocations, setCareLocations] = React.useState([]);
```

Comment:
Could you move line 11 - 19 to a separate hook? And update `useState` and `useEffect` instead of React.

Reply:
Hey Karthikk, thanks for the feedback!

Sorry, could you please clarify what you mean by "move this to a separate hook", I am kind of new to this? As in, move into a separate file and then import this whole functionality or? Thanks in advance!

Reply:
Move it to a separate file under hooks or services as useCareLocation(), and export the `careLocations` variable alone from the hook and import it in this file